### PR TITLE
 [CLOSES] 2429 [Term Entry] JavaScript Dates .setFullYear() 

### PR DESCRIPTION
**Pull request by Yashi11**
### Description
A new term entry for JavaScript [Dates](https://www.codecademy.com/resources/docs/javascript/dates) about the .setFullYear() method in a new file in docs/content/javascript/concepts/dates/terms/setFullYear/setFullYear.md with  an introductory paragraph about what the method is and does , an example showing the method in use and others as per the requirement.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setFullYear/setFullYear.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+Title: '.setFullYear()'
+Description: 'Changes the year, month, and/or day of month  according to local time.'
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setFullYear/setFullYear.md:3]:
```suggestion
Description: 'Changes the year, month, and/or day according to local time.'
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Yashi11 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/javascript/concepts/dates/terms/setFullYear/setFullYear.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```js
+const exampleDate = new Date();
+exampleDate.setFullYear(1887,7);
+console.log(exampleDate);
+```
+
+This example results in the following output:
+
+```shell
+Sun Aug 14 1887 18:21:27 GMT+0521 (India Standard Time)
```
[the remark this answers is not in the document]
Comment (Yashi11) [reply content/javascript/concepts/dates/terms/setFullYear/setFullYear.md:40]: This is the output that I have received after running the required code only i.e. the exact code mentioned above the shell output, still I will re-check it once and update accordingly.

**Comment by Yashi11** [comment]
@caupolicandiaz Done with the required changes 👍

**Review by caupolicandiaz — APPROVED** [review]
@Yashi11 This is ready for a second review.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yashi11 I did a second review and had a few comments.

**Comment by Yashi11** [comment]
Done with the required changes.

**Review by SSwiniarski — APPROVED** [review]
@Yashi11 Looks good for a second review.

**Pull request merged**